src/Domain/Tool/Phpunit/PhpUnitTask.php

```diff
@@ -324,7 +324,7 @@ protected function runPhpUnit(): void {
         '--verbose',
       ];
       if ($this->shouldGenerateCodeCoverage()) {
-        $command[] = "--coverage-cobertura={$this->cloverCoverage}";
+        $command[] = "--coverage-cobertura={$this->coberturaCoverage}";
```

Comment:
We still need `--coverage-cobertura` _and_ `--coverage-clover`.  Can we generate both on the same run? If not, we'll need to determine which to generate based on environment variables.

Reply:
Yeah, I will create separate environment variables like by default it will generate coverage in CLOVER but if `ORCA_COVERAGE_COBERTURA_ENABLE` is set then it will generate coverage in COBERTURA.

Reply:
Done.